portfolio/src/main/webapp/script.js

```diff
@@ -83,33 +83,46 @@ window.addEventListener('scroll', function() {
     document.getElementById('navbar').style.padding = '30px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 0.25)';
-    document.getElementById('myName').style.fontSize = '25px';
+    document.getElementById('navbar-name').style.fontSize = '25px';
   }
   else {
     document.getElementById('navbar').style.padding = '80px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 1)';
-    document.getElementById('myName').style.fontSize = '35px';
+    document.getElementById('navbar-name').style.fontSize = '35px';
   }
 })
 
 /**
- * Fetches all messages (submitted via the form) from the server
+ * Fetches a number of comments (submitted via the form) from the server
  * and adds it to the DOM.
  */
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((messages) => {
-    const element = document.getElementById('message-container')
-    element.innerHTML = '';
-    for (let key in messages) {
-      element.appendChild(createListElement(messages[key]));
-    }
+function getComments() {
+  let url = '/data?limit=' + document.getElementById("quantity").value.toString();
+  fetch(url).then(response => response.json()).then((comments) => {
+    const element = document.getElementById('comment-container');
+    element.innerHTML = "";
+    comments.forEach((comment) => {
+      console.log(comment.content);
+      element.appendChild(createCommentElement(comment));
+    })
   });
 }
 
 /** Creates an <li> element containing text. */
```

Comment:
Can you update this comment?

Reply:
Yup! I added the return tag, and wrote 'element' inside the brackets for the return type. Is that ok?

Reply:
I think what you want is something more like 'const commentContainer = document.getElementById('comment-container')' - it's clear what it is at the declaration but the name is important so it's clear wherever it's used what exactly that element's purpose is.

Reply:
Done!